.github/workflows/vscan.yaml

```diff
@@ -0,0 +1,42 @@
+# A workflow to create the Vulnerability Scan Report of the input image 
+# that may be (unoptimized) version of the image initally built
+# or the hardened image.
+#
+# The current workflow:
+#  - Installs slimctl.
+#  - Creates the Vulnerability Scan Report of the input image
+
+name: vuln-scan
+on:
+  workflow_call:
+    inputs:
+      # The image whose vuln scan report will be created.
+      image:
+        required: true
+        type: string
+
+jobs:
+  vuln-scan-report:
```

Comment:
Let's call this job "vulnerability-scan", because the report is just one of the artifacts it may produce (we also will have the history of calls and a nicely looking UI as other types of artifacts in the future).